### PR TITLE
RSS Feed - add user-agent header

### PR DESCRIPTION
**Pull request by adi88d**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-41991)

## Description
- added the `User-Agent` header to simulate request from browser.
- save the `channel.link` value in case the `item.link` is not contains the website url. 

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py [hunk 1]
```diff
@@ -40,11 +44,21 @@ def parse_feed_data(self, feed_response):
...
+
         parsed_indicators: list = []
         if not self.feed_data:
             raise DemistoException(f"Could not parse feed data {self._base_url}")
 
         for indicator in reversed(self.feed_data.entries):
+
+            link = indicator.get('link')
+            if link and not link.startswith(('http://', 'https://')):
+                link = urljoin(self.channel_link, link)
```
Comment (DeanArbel) [inline Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py:60]: Please add a debug message here

**Review by DeanArbel — APPROVED** [review]
Please add the debug message, and add a release note about using the `channel` as a root for feeds that require it as a prefix.

File: Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py (hunk 1 above)
Comment (adi88d) [reply Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py:61]: done

File: Packs/FeedRSS/ReleaseNotes/1_0_13.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### RSS Feed
+
+- Fixed an issue where the **fetch-indicators** command failed to fetch indicators from some feeds.
```
Comment (ShirleyDenkberg) [inline Packs/FeedRSS/ReleaseNotes/1_0_13.md:6]:
```suggestion
- Fixed an issue where the ***fetch-indicators*** command failed to fetch indicators from some feeds.
```

**Comment by ShirleyDenkberg** [comment]
@DeanArbel Doc review completed.

**Pull request merged**